### PR TITLE
Convert CommonJS to ES6 syntax, correct typos and mismatched words

### PR DESCRIPTION
**Pull request by BaharaJr**
Hey chief, I looked into the words.json file and tried to make a few corrections

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -86,3 +86,6 @@ typings/
 
 # DynamoDB Local files
 .dynamodb/
+
+# GitHub files
+.github
```
Comment (alphaolomi) [inline .gitignore]: Why remove .github files?

**Review by alphaolomi — COMMENTED** [review]
Umetisha @BaharaJr

File: .gitignore (hunk 1 above)
Comment (BaharaJr) [reply .gitignore]: So when I clone, it comes  .github files that I would not want to push to the remote with the same files

Comment (BaharaJr) [reply .gitignore]: > Why remove .github files?

So I figured I wasn't supposed to remove them

Comment (alphaolomi) [reply .gitignore]: 🤝

**Pull request merged**